.github/workflows/ngtcp2-linux.yml

```diff
@@ -51,7 +51,7 @@ env:
   ngtcp2-version: v0.18.0
   nghttp2-version: v1.55.1
   gnutls-version: 3.8.0
-  wolfssl-version: v5.6.3-stable
+  wolfssl-version: master
```

Comment:
Does this make the caching weird if we cache on master?

Reply:
> Does this make the caching weird if we cache on master?

Indeed, need to take that out. Thanks for noticing.